configure/CONFIG_COMMON

```diff
@@ -81,6 +81,11 @@ FINAL_LOCATION              = $(INSTALL_ABSOLUTE)
 # IOC's view of install path
 IOCS_APPL_TOP               = $(INSTALL_ABSOLUTE)
 
+#-------------------------------------------------------
+# Temporary build products
+
+GENERATED_MARKDOWN_LOCATION = $(TOP)/documentation/generated
```

Comment:
I'm still not keen on this semi-installation directory path. Maybe using the `$(INSTALL_DOC)` path would be appropriate and wouldn't need as many additions to the configure files. Markdown is now a common documentation file format, all-be-it more normally used as a source format.

Reply:
That would mean quite a bit refactor, right now. These Markdown file are installed in this private directory because they need to be included during the RTD build, whose root is `$(TOP)/documentation`. I won't have time to rethink that architecture in the following months, unfortunately…